content/apply/index.en.md

```diff
@@ -17,41 +17,41 @@ We help NGOs to make use of their data by connecting them with volunteer data sc
 <div id="apply--what">
     <ul>
         <li>
-            <a href="{{< ref "datenanalyse-fuer-den-guten-zweck#zielgruppen-verstehen" >}}">
+            <a href="{{< ref "datenanalyse-fuer-den-guten-zweck#zielgruppen-verstehen" lang="en">}}">
```

Comment:
Why is this needed?

Reply:
I assumed that we eventually want to translate the use cases. Otherwise, what is the point to link them from here?

Reply:
Yep, but there are already files for both translations. It's just german in both versions.

Reply:
See https://github.com/dssg-berlin/dssg-berlin.org/tree/master/content/blog

Reply:
Yes. Please see my comment in Slack. Maybe I misunderstood how the translations work

Reply:
Yes, not your fault. It's confusing.